### PR TITLE
Added expression and statement roles

### PR DESCRIPTION
**Pull request by juanjux**
Similar to bblfsh/java-driver#33 after agreeing that explicit is better.

### CODE REVIEW

File: ANNOTATION.md [hunk 1]
```diff
@@ -33,109 +33,109 @@
...
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@InternalType='Compare\.comparators'\] | BinaryExpressionRight |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@internalRole\]\[@internalRole='left'\] | BinaryExpressionLeft |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If, Expression |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If, Expression, Statement |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\]/\*\[@internalRole\]\[@internalRole='body'\] | IfBody |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\]/\*\[@internalRole\]\[@internalRole='test'\] | IfCondition |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\]/\*\[@internalRole\]\[@internalRole='orelse'\] | IfElse |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If, Statement |
```
Comment (abeaumont) [inline ANNOTATION.md]: This looks like a duplicate of line 102.

File: ANNOTATION.md [hunk 2]
```diff
@@ -33,109 +33,109 @@
...
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@internalRole\]\[@internalRole='test'\] | IfCondition |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='If\.orelse'\] | IfElse |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\] | BinaryExpression |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\] | BinaryExpression, Expression |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@InternalType='Compare\.ops'\] | BinaryExpressionOp |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@InternalType='Compare\.comparators'\] | BinaryExpressionRight |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='If'\]/\*\[@InternalType='Compare'\]/\*\[@internalRole\]\[@internalRole='left'\] | BinaryExpressionLeft |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If, Expression |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='IfExp'\] | If, Expression, Statement |
```
Comment (abeaumont) [inline ANNOTATION.md]: Is this both an expression and a statement?

File: ANNOTATION.md [hunk 3]
```diff
@@ -33,109 +33,109 @@
...
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Print'\] | Call, CallCallee, SimpleIdentifier, Expression |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Print'\]/\*\[@internalRole\]\[@internalRole='dest'\] | CallPositionalArgument |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Print'\]/\*\[@internalRole\]\[@internalRole='nl'\] | CallPositionalArgument |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Print'\]/\*\[@internalRole\]\[@internalRole='values'\] | CallPositionalArgument |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Print'\]/\*\[@internalRole\]\[@internalRole='values'\]/\*\[\*\] | CallPositionalArgument |
 | /self::\*\[@InternalType='Module'\]//\*\[@internalRole\]\[@internalRole='annotation'\] | Comment |
 | /self::\*\[@InternalType='Module'\]//\*\[@internalRole\]\[@internalRole='returns'\] | Comment |
 | /self::\*\[@InternalType='Module'\]//\*\[@InternalType='Ellipsis'\] | SimpleIdentifier |
-| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='comprehension'\] | ForEach |
+| /self::\*\[@InternalType='Module'\]//\*\[@InternalType='comprehension'\] | ForEach, Expression, Statement |
```
Comment (abeaumont) [inline ANNOTATION.md]: Is a list comprehension a statement?

File: ANNOTATION.md (hunk 2 above)
Comment (juanjux) [inline ANNOTATION.md]: I tough so, but it seems it's only an expression (https://help.semmle.com/wiki/display/QL/Tutorial%3A+Statements+and+Expressions).

File: ANNOTATION.md (hunk 3 above)
Comment (juanjux) [inline ANNOTATION.md]: Only expression, the statement would be the `for` inside (trough the `if` inside the comprehensions is given by the Python AST just as a `compare` which is an Expression, but also containing the `if` keyword I've doubts if that one could be considered both).

**Pull request merged**